W-22436930-omni-governance-sections (#416)
mulesoft/docs-general: modules/ROOT/pages/exp-view-cost-and-token-usage.adoc

> = View Cost and Token Usage

Reviewer: ```suggestion
= Viewing Cost and Token Usage
```

Reviewer: gerund for multi-task topics

Use cost and token metrics to understand usage, control spend, and identify optimization opportunities.

Available metrics depend on your tenant configuration and role.

> == Where Token and Performance Signals Appear

Reviewer: ```suggestion
== Cost and Token Data Locations
```

> The system surfaces token usage and performance data in several places:

Reviewer: ```suggestion
The system displays token usage and performance data in several places:
```

> * *Governance* > *Cost Management*: Primary dashboard for organization wide token usage and spend indicators.

Reviewer: ```suggestion
* *Governance* > *Cost Management*: Primary dashboard for organization-wide token usage and spend indicators
```

> * Service detail pages: Usage and performance summaries for supported services.

Reviewer: ```suggestion
* Service detail pages: Usage and performance summaries for supported services
```

> * *Observability*: Aggregated token and latency trends, when configured.

Reviewer: ```suggestion
* *Observability*: Aggregated token and latency trends, when configured
```

Navigation labels can vary by release.

== Open Cost Management

. Log in and go to *Governance* > *Cost Management*.
. Adjust the time range and filters for the services or period you want to analyze.

> == Understand Usage Metrics

Reviewer: ```suggestion
== Usage Metrics
```

Reviewer: Don't use a verb for non-task topic headings

The experience organizes token and activity data into summary cards and detail views:

> * *Total tokens*: Estimated tokens processed in the selected time range.

Reviewer: ```suggestion
* *Total tokens*: Number of tokens processed in the selected time range
```

> * *Saved tokens, estimated*: Estimated tokens saved by optimization policies.

Reviewer: ```suggestion
* *Saved tokens, estimated*: Estimated tokens saved by optimization policies
```

> * *Tool calls*: Number of tool or function calls in the selected time range.

Reviewer: ```suggestion
* *Tool calls*: Number of tool or function calls in the selected time range
```

Reviewer: I wonder if we should use Title Case for these, since they're headings. Even though they're ALL CAPS in the UI.

Each card also shows how many instances contributed to the metric.

== Filter and Search

Use controls at the top of the page to narrow the view:

* Search for service or instance names.
* Use *Filters* to refine your search by service type, environment, or metadata.
* Select 1H, 6H, 24H, 7D, or 30D to search per time range.

The counter updates to show the number of matching services.

> == Interpret Saved Tokens

Reviewer: ```suggestion
== Saved Token Estimates
```

When optimization policies are active, the system estimates saved tokens compared to unoptimized traffic.

Treat this metric as directional, not exact.

> == How You Use the Data

Reviewer: ```suggestion
== Cost Data Use Cases
```

Cost and performance data inform multiple workflows:

* Cost optimization: Identify services with high usage and apply optimization controls.
* Performance triage: Correlate latency or error spikes with deployments or policy changes.
* Planning and alignment: Share trends with finance and platform teams for forecasting.

> This dashboard does't automatically cap usage. To limit consumption, apply governance strategies or rate limiting policies.

Reviewer: ```suggestion
This dashboard doesn't automatically cap usage. To limit consumption, apply governance strategies or rate-limiting policies.
```

> == Where Cost Fits Into Governance

Reviewer: ```suggestion
== Cost Data and Governance 
```
